<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetMemo</class>
 <widget class="QWidget" name="widgetMemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widgetMemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <spacer name="leftSpace">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox { color: white; background-color:rgb(227, 227, 227); border-width:0px; border-radius:6px;} QGroupBox:hover { background-color:rgb(170, 170, 255); }</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblUserName">
          <property name="styleSheet">
           <string notr="true">QLabel
{
color:black;
background-color:transparent;
font:bold;
font-size:8pt;
}</string>
          </property>
          <property name="text">
           <string>사용자</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMemo">
          <property name="styleSheet">
           <string notr="true">QLabel
{
color:black;
background-color:transparent;
font:bold;
font-size:12pt;
text-align:left;
}</string>
          </property>
          <property name="text">
           <string>여기에 메모가 들어감
여러 줄이 들어감</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDateTime">
          <property name="styleSheet">
           <string notr="true">QLabel
{
color:black;
background-color:transparent;
font:bold;
font-size:8pt;
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
